src/main/webapp/app/exam/overview/events/button/exam-live-events-button.component.ts

```diff
@@ -32,24 +32,27 @@ export class ExamLiveEventsButtonComponent implements OnInit, OnDestroy {
     private liveEventsSubscription?: Subscription;
     private allEventsSubscription?: Subscription;
     eventCount = 0;
-    @Input() examStartDate: dayjs.Dayjs;
+    examStartDate = input<dayjs.Dayjs>();
```

Comment:
The component does not work properly without this value, does it?
```suggestion
    examStartDate = input.required<dayjs.Dayjs>();
```

Reply:
Good point, thanks! It wasn't required because it was used in `exam-navigation-bar.component.html`, where this parameter wasn't defined. However, it was redundant. I removed the component from `exam-navigation-bar.component.html` and made `examStartDate` required.